### PR TITLE
added flag fail-if-troubled

### PR DESCRIPTION
**Pull request by daimor**
I've added flag `-fail-if-troubled` as discussed in #6

### CODE REVIEW

File: ccontainermain.go [hunk 1]
```diff
@@ -483,6 +485,7 @@ func main() {
 	pFnostu := flag.Bool("nostu", false, "Allows cstart to run with the nostu option for maintenance, single user access mode.")
 	pFshmem := flag.Int("shmem", 512, "Shared Mem segment max size in MB; default value=512MB enough to install and play")
 	pFlog := flag.Bool("cconsole", false, "Allows to show cconsole.log in current output.")
+	pFail := flag.Bool("fail-if-troubled", false, "Container will fail if server does not start correctly")
```
Comment (zrml) [inline ccontainermain.go]: The description should be: "Container will fail if Caché does not start correctly"

File: ccontainermain.go [hunk 2]
```diff
@@ -269,7 +271,7 @@ func shutdownCaché(inst string) (bool, error) {
 	}
 
 	// check that the shutdown was successful
-	if err := checkCmdOutcome("down", inst); err != nil {
+	if err := checkCmdOutcome("down", inst, false); err != nil {
 		log.Printf("Error: Caché was not shutdown successfully.\n")
```
Comment (zrml) [inline ccontainermain.go:275]: in the log.Printf() we need to review the comment. Not sure we can capture all possible cases. Maybe we should return the parsed info string...

**Review by zrml — COMMENTED** [review]
please review my comments

File: ccontainermain.go (hunk 2 above)
Comment (daimor) [reply ccontainermain.go:275]: Good idea, I'll try to find a way how to do it.

Comment (zrml) [reply ccontainermain.go:275]: OK Thanks @daimor. If i remember correctly I already parse the string for the relevant message. Either send the whole string before parsing or the parsed part... you'll have to change functions signature... Thanks

**Review by zrml — APPROVED** [review]

**Pull request merged**